project-management/src/main/java/life/qbic/projectmanagement/application/api/AsyncProjectServiceImpl.java

```diff
@@ -768,40 +767,48 @@ private Mono<ValidationResponse> validateMetadata(
   }
 
   private Mono<ValidationResponse> validateMeasurementMetadataPxP(
-      MeasurementRegistrationInformationPxP registration, String requestId, String projectId) {
+      MeasurementRegistrationInformationPxP registration, String requestId, String experimentId,
+      String projectId) {
     var securityContext = SecurityContextHolder.getContext();
     return applySecurityContext(Mono.fromCallable(
-            () -> measurementValidationService.validatePxp(registration, ProjectId.parse(projectId)))
+            () -> measurementValidationService.validatePxp(registration, experimentId,
+                ProjectId.parse(projectId)))
         .map(validationResult -> new ValidationResponse(requestId, validationResult)))
         .contextWrite(reactiveSecurity(securityContext))
         .subscribeOn(scheduler);
   }
 
   private Mono<ValidationResponse> validateMeasurementMetadataPxPUpdate(
-      MeasurementUpdateInformationPxP update, String requestId, String projectId) {
+      MeasurementUpdateInformationPxP update, String requestId, String experimentId,
+      String projectId) {
     var securityContext = SecurityContextHolder.getContext();
     return applySecurityContext(Mono.fromCallable(
-            () -> measurementValidationService.validatePxp(update, ProjectId.parse(projectId)))
+            () -> measurementValidationService.validatePxp(update, experimentId,
```

Comment:
Some reformatting again without actual changes :/

Reply:
Not correct 😄 `experimentId` was added which leads to a longer line that is broken as a result of that.